### PR TITLE
Copy changes

### PR DESCRIPTION
**Pull request by levb**

### CODE REVIEW

**Review by aaronrothschild — APPROVED** [review]
Thanks @levb for putting this in the proper branch.

File: server/setup_flow.go [hunk 1]
```diff
@@ -101,18 +101,18 @@ func continueButton(next flow.Name) flow.Button {
 
 func (p *Plugin) stepWelcome() flow.Step {
 	return flow.NewStep(stepSetupWelcome).
-		WithPretext("##### :wave: Welcome to Jira integration! [Learn more](https://github.com/mattermost/mattermost-plugin-jira#readme)").
-		WithTitle("Configure the integration.").
-		WithText("Just a few steps to go!\n" +
+		WithPretext("##### :wave: Welcome to the Jira integration! [Learn more](https://github.com/mattermost/mattermost-plugin-jira#readme)").
+		WithTitle("Let's Configure the connection with Jira.").
+		WithText("There's just a few steps!\n" +
 			"1. Choose the Jira edition (cloud or server) you will connect to.\n" +
```
Comment (cwarnermm) [inline server/setup_flow.go:107]:
```suggestion
			"1. Choose the Jira edition (cloud or server) you'll connect to.\n" +
```

File: server/setup_flow.go [hunk 2]
```diff
@@ -101,18 +101,18 @@ func continueButton(next flow.Name) flow.Button {
...
-			"2. Configure the Mattermost integration (app) in Jira.\n" +
+			"2. Configure the Mattermost integration (app) within Jira.\n" +
```
Comment (cwarnermm) [inline server/setup_flow.go:108]: I'm unclear about what "(app)" is communicating above. Perhaps "Configure the Mattermost Jira integration in Jira." would suffice?

File: server/setup_flow.go [hunk 3]
```diff
@@ -101,18 +101,18 @@ func continueButton(next flow.Name) flow.Button {
...
 			"3. Configure the subscriptions webhook in Jira.\n" +
 			"4. Connect your user account.\n" +
-			"Configuring the integration requires administrator access to Jira. If you aren't a Jira admin, " +
-			"select **I need someone else** to ask another Mattermost user to do it.\n" +
+			"Note: Configuring the integration requires administrator access to Jira. If you aren't a Jira admin, " +
+			"select **I need someone else** to ask another Mattermost user to perform this configuration. \n" +
```
Comment (cwarnermm) [inline server/setup_flow.go:112]:
```suggestion
			"select **I need someone else** to ask another Mattermost user to complete this configuration. \n" +
```

File: server/setup_flow.go [hunk 4]
```diff
@@ -101,18 +101,18 @@ func continueButton(next flow.Name) flow.Button {
...
 			"\n" +
-			"You can **Cancel** these steps at any time and use `/jira` command to finish the configuration later. " +
-			"See [documentation](https://mattermost.gitbook.io/plugin-jira/setting-up/configuration) for more.").
+			"You can **Cancel** these steps at any time and use `/jira` command to complete the configuration later. " +
+			"See [documentation](https://mattermost.gitbook.io/plugin-jira/setting-up/configuration) for more details.").
```
Comment (cwarnermm) [inline server/setup_flow.go:115]:
```suggestion
			"See [documentation](https://mattermost.gitbook.io/plugin-jira/setting-up/configuration) for details.").
```

File: server/setup_flow.go [hunk 5]
```diff
@@ -143,7 +143,7 @@ func (p *Plugin) stepWelcome() flow.Step {
 
 func (p *Plugin) stepDelegated() flow.Step {
 	return flow.NewStep(stepDelegated).
-		WithText("Asked {{.Delegated}} to finish configuring the integration.").
+		WithText("Asked {{.Delegated}} to finish configuring the integration. They will get a notification and be able to go through the wizard.").
```
Comment (cwarnermm) [inline server/setup_flow.go:146]:
```suggestion
		WithText("Asked {{.Delegated}} to finish configuring the integration. They will receive a notification to complete the configuration.").
```

File: server/setup_flow.go [hunk 6]
```diff
@@ -208,10 +208,10 @@ func (p *Plugin) stepChooseEdition() flow.Step {
 
 func (p *Plugin) stepServerAddAppLink() flow.Step {
 	return flow.NewStep(stepServerAddAppLink).
-		WithPretext("##### :white_check_mark: Step 2: Configure Mattermost Application Link in Jira").
+		WithPretext("##### :white_check_mark: Step 2: Configure the Mattermost Application Link in Jira").
```
Comment (cwarnermm) [inline server/setup_flow.go:211]: This step 2 text should match the one on line 108 (assuming they're the same step). With this in mind, which version of step 2 text is the one we'll keep? Which one is more representative and describes what the user is trying to achieve?

File: server/setup_flow.go [hunk 7]
```diff
@@ -235,7 +235,7 @@ func (p *Plugin) stepServerConfirmAppLink() flow.Step {
 func (p *Plugin) stepServerConfigureAppLink1() flow.Step {
 	return flow.NewStep(stepServerConfigureAppLink1).
 		WithTitle("Create Incoming Application Link.").
-		WithText("Complete the following steps, then come back here and select **Continue**.\n\n"+
+		WithText("Complete the following steps in Jira, then come back here and select **Continue**.\n\n"+
```
Comment (cwarnermm) [inline server/setup_flow.go:238]:
```suggestion
		WithText("Complete the following steps in Jira, then come back here to select **Continue**.\n\n"+
```

File: server/setup_flow.go [hunk 8]
```diff
@@ -235,7 +235,7 @@ func (p *Plugin) stepServerConfirmAppLink() flow.Step {
...
 			"1. In Jira's **Link Applications** screen (see _screenshot_) enter the following values, leave all other fields blank.\n"+
```
Comment (cwarnermm) [inline server/setup_flow.go:239]:
```suggestion
			"1. In Jira's **Link Applications** screen (see _screenshot_) enter the following values, and leave all other fields blank.\n"+
```

File: server/setup_flow.go [hunk 9]
```diff
@@ -249,7 +249,7 @@ func (p *Plugin) stepServerConfigureAppLink1() flow.Step {
 func (p *Plugin) stepServerConfigureAppLink2() flow.Step {
 	return flow.NewStep(stepServerConfigureAppLink2).
 		WithTitle("Configure Incoming Application Link.").
-		WithText("Complete the following steps, then come back here and select **Continue**.\n\n"+
+		WithText("Complete the following steps in Jira, then come back here and select **Continue**.\n\n"+
```
Comment (cwarnermm) [inline server/setup_flow.go:252]:
```suggestion
		WithText("Complete the following steps in Jira, then come back here to select **Continue**.\n\n"+
```

File: server/setup_flow.go [hunk 10]
```diff
@@ -155,23 +155,23 @@ func (p *Plugin) stepDelegated() flow.Step {
...
-		WithText("Please choose whether you use the Jira Cloud or Server (on-premise) edition. " +
+		WithText("Please choose whether you use the Jira Cloud or Server (on-premise/DC) edition. " +
 			"If you need to integrate with more than one Jira instance, please refer to the [documentation](https://mattermost.gitbook.io/plugin-jira/)").
 		WithButton(flow.Button{
 			Name:  "Jira Cloud",
 			Color: flow.ColorPrimary,
 			Dialog: &model.Dialog{
 				Title:            "Enter Jira Cloud Organization",
-				IntroductionText: "Enter Jira Cloud URL (usually, `https://yourorg.atlassian.net`), or just the organization part, `yourorg`.",
+				IntroductionText: "Enter Jira Cloud URL (usually, `https://yourorg.atlassian.net`), or just the organization part, `yourorg`",
```
Comment (cwarnermm) [inline server/setup_flow.go:174]:
```suggestion
				IntroductionText: "Enter a Jira Cloud URL (typically, `https://yourorg.atlassian.net`), or just the organization part, `yourorg`",
```

File: server/setup_flow.go [hunk 11]
```diff
@@ -268,11 +268,11 @@ func (p *Plugin) stepCloudAddedInstance() flow.Step {
 
 func (p *Plugin) stepCloudEnableDeveloperMode() flow.Step {
 	return flow.NewStep(stepCloudEnableDeveloperMode).
-		WithPretext("##### :white_check_mark: Step 2: configure the Mattermost app in Jira").
+		WithPretext("##### :white_check_mark: Step 2: Configure the Mattermost app in Jira").
```
Comment (cwarnermm) [inline server/setup_flow.go:271]: Is this the same step as line 108 and 211?

File: server/setup_flow.go [hunk 12]
```diff
@@ -268,11 +268,11 @@ func (p *Plugin) stepCloudAddedInstance() flow.Step {
...
 		WithTitle("Enable development mode.").
 		WithText("Mattermost Jira Cloud integration requires setting your Jira to _development mode_ (see _screenshot_). "+
 			"Enabling the development mode allows you to install apps like Mattermost, from outside the Atlassian Marketplace.\n"+
-			"Complete the following steps, then come back here and select **Continue**.\n\n"+
+			"Complete the following steps in Jira, then come back here and select **Continue**.\n\n"+
```
Comment (cwarnermm) [inline server/setup_flow.go:275]:
```suggestion
			"Complete the following steps in Jira, then come back here to select **Continue**.\n\n"+
```

File: server/setup_flow.go [hunk 13]
```diff
@@ -314,7 +314,8 @@ func (p *Plugin) stepInstalledJiraApp() flow.Step {
 	}
 	return flow.NewStep(stepInstalledJiraApp).
 		WithTitle("Confirmed.").
-		WithText("You've finished configuring the Mattermost App in Jira. Select **Continue** to setup the subscription webhook.").
+		WithText("You've finished configuring the Mattermost App in Jira. Select **Continue** to setup the subscription webhook" +
```
Comment (cwarnermm) [inline server/setup_flow.go:317]:
```suggestion
		WithText("You've finished configuring the Mattermost App in Jira. Select **Continue** to set up the subscription webhook" +
```

File: server/setup_flow.go [hunk 14]
```diff
@@ -331,7 +332,7 @@ func (p *Plugin) stepInstalledJiraApp() flow.Step {
 func (p *Plugin) stepWebhook() flow.Step {
 	return flow.NewStep(stepWebhook).
 		WithPretext("##### :white_check_mark: Step 3: Setup Jira Subscriptions Webhook").
-		WithText(`To receive Jira event notifications in Mattermost you need to set up a single "firehose" `+
+		WithText(`To receive Jira event notifications in Mattermost Channels you need to set up a single "firehose" `+
```
Comment (cwarnermm) [inline server/setup_flow.go:335]:
```suggestion
		WithText(`To receive Jira event notifications in Mattermost Channels, you need to set up a single "firehose" `+
```

File: server/setup_flow.go [hunk 15]
```diff
@@ -363,8 +364,8 @@ func (p *Plugin) stepWebhook() flow.Step {
 
 func (p *Plugin) stepWebhookDone() flow.Step {
 	return flow.NewStep(stepWebhookDone).
-		WithTitle("Webhook is setup.").
-		WithText("You can now use `/jira subscribe` from a specific channel to receive change notifications from Jira there.").
+	WithTitle("Webhook is setup. :tada:").
+		WithText("You can now use the command `/jira subscribe` from a specific channel to receive notifications from Jira there.").
```
Comment (cwarnermm) [inline server/setup_flow.go:368]:
```suggestion
		WithText("You can now use the command `/jira subscribe` from a specific channel to receive Jira notifications in that channel.").
```

**Review by cwarnermm — COMMENTED** [review]
Thanks! Feedback provided inline. Please let me know if there are any questions or concerns.

File: server/setup_flow.go [hunk 16]
```diff
@@ -155,23 +155,23 @@ func (p *Plugin) stepDelegated() flow.Step {
...
 		Next(stepDone)
 }
 
 func (p *Plugin) stepChooseEdition() flow.Step {
 	return flow.NewStep(stepChooseEdition).
-		WithPretext("##### :white_check_mark: Step 1: Choose Jira Edition.").
+		WithPretext("##### :white_check_mark: Step 1: Which Jira edition do you use?").
 		WithTitle("Cloud or Server (on-premise).").
-		WithText("Please choose whether you use the Jira Cloud or Server (on-premise) edition. " +
+		WithText("Please choose whether you use the Jira Cloud or Server (on-premise/DC) edition. " +
```
[the remark this answers is not in the document]
Comment (levb) [reply server/setup_flow.go:178]: -> Data Center

File: server/setup_flow.go (hunk 11 above)
Comment (levb) [reply server/setup_flow.go:282]: See my comment on line 211, same conceptually, different technically.

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**